lib/presentation/crowdaction/crowdaction_comments/parts/comment_appbar_delegate.dart

```diff
@@ -0,0 +1,78 @@
+part of '../crowdaction_comments_page.dart';
+
+class CommentAppBarDelegate extends SliverPersistentHeaderDelegate {
+  final double expandedHeight;
+  final Color color;
+
+  CommentAppBarDelegate({required this.expandedHeight, required this.color});
+
+  @override
+  Widget build(
+    BuildContext context,
+    double shrinkOffset,
+    bool overlapsContent,
+  ) {
+    return Stack(
+      clipBehavior: Clip.none,
+      fit: StackFit.expand,
+      children: [
+        ColoredBox(color: color),
+        if (shrinkOffset < 100) ...[
+          _buildHeaderContent(),
```

Comment:
The content of this method can be a `const StatelessWidget` therefore it shouldn't be a method. It will keep calling this method whenever this widget rebuilds.